.pre-commit-config.yaml

```diff
@@ -10,9 +10,7 @@ repos:
     rev: 3.7.9
     hooks:
     - id: flake8
-      args:
-        - "--max-line-length=100"
-        - "--ignore=E203,E266,E501,W503"
+      exclude: .*\.t
```

Comment:
Why exclude .t from linter if they still are python files?

Reply:
Because they've been copied from elsewhere and they don't comply with flake8 :)